examples/user_managed_deps/package.json

```diff
@@ -3,6 +3,7 @@
   "devDependencies": {
     "jasmine": "~3.4.0",
     "jasmine-core": "~3.4.0",
+    "jasmine-reporters": "~2.3.2",
```

Comment:
the fact that you had to update this spot suggests it's a breaking change - I'd rather just silently not produce the xml output if we don't find jasmine-reporters

Reply:
Yup, that sounds good. I had not considered this style of dep management when I initially made the PR and stated that it was not breaking. I updated to remove this change and to make the reporters fail silently